common/Util.hpp

```diff
@@ -1300,6 +1300,8 @@ int main(int argc, char**argv)
      */
     bool isFuzzing();
 
+    bool isKitInProcess();
```

Comment:
shouldn't this me `const` ?

Reply:
just a function declaration so i think it is okay :)

Reply:
duh I misread the scope. nevermind.